requirements.txt

```diff
@@ -3,4 +3,7 @@ numpy>=1.3
 pandas>=1
 networkx
 libpysal
-ortools
\ No newline at end of file
+ortools
+sklearn
```

Comment:
Ah! This should be `scikit-learn`! (See[ here](https://scikit-learn.org/stable/install.html#installation-instructions)). I am always confused about `scikit-learn`'s install name and import alias...

Reply:
Is it possible due to the [version of python](https://stackoverflow.com/questions/36404042/importerror-no-module-named-sklearn-python)?

Reply:
It may be, but I am still not sure why the environment seems to not even attempt to download `scikit-learn`. See environment list [here](https://github.com/pysal/spopt/pull/65/checks?check_run_id=1059112512#step:3:832). I went through the `Run goanpeca/setup-miniconda@v1` process and did not even see any mention of `scikit-learn`. However, `geopandas` (which you just added) is being installed. This seems to be strange, but it will most likely be something small we overlooked... I have to work on some other stuff now, but maybe @knaaptime can give some advice?

Reply:
[`conda-forge/scikit-learn`](https://github.com/conda-forge/scikit-learn-feedstock#current-build-status) is passing on all platforms, so it does not appear to be version/Python issue.

Reply:
> This seems to be strange, but it will most likely be something small we overlooked... 

This ended up being true. It was an oversight of mine... see fe8dbcd